code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -175,6 +175,8 @@ def __load_config(self, **kwargs) -> None:
         self.ORCHESTRATION_STRATEGY = os.getenv(
             "ORCHESTRATION_STRATEGY", "openai_function"
         )
+        # Ensure this environment variable is set
+        os.environ["ORCHESTRATION_STRATEGY"] = self.ORCHESTRATION_STRATEGY
```

Comment:
Is it worth writing a test to verify this? Is there any check elsewhere that will catch the deletion of this line 179, since otherwise the json file will not get this value?

Reply:
Good idea

Reply:
So I reworked it to still use the env helper instead of relying on env vars directly, hopefully this is easier to maintain